plugins/all-modules-page/src/test/kotlin/templates/ResolveLinkCommandResolutionTest.kt

```diff
@@ -56,7 +56,7 @@ class ResolveLinkCommandResolutionTest : MultiModuleAbstractTest() {
             }
         }
 
-        val contentFile = setup(link)
+        val contentFile = setup(link, testedDri)
```

Comment:
Could you add a test case for [the case](https://github.com/Kotlin/dokka/issues/2272#issuecomment-997076375) from #2272?